modules/manage/pages/manage-security/manage-connections-and-disks.adoc

```diff
@@ -142,4 +143,4 @@ If _file-level_ is chosen, the following Couchbase directories and files should
  ** Windows: `C:\Program Files\couchbase\server\var\lib\couchbase\isasl.pw` and `C:\Program Files\couchbase\server\var\lib\couchbase\var\lib\config\`.
 
 For more information, see the webinar provided at
-http://www.couchbase.com/nosql-resources/webinar/recorded[Understanding Database Encryption with Couchbase and Vormetric].
+{url-webinar}[Understanding Database Encryption with Couchbase and Vormetric].
```

Comment:
```suggestion
{url-webinar}[Understanding Database Encryption with Couchbase & Vormetric^].
```
Updated link text to match exact webinar title; added caret so link opens in new tab